app/src/main/java/xyz/jekyllex/utils/Settings.kt

```diff
@@ -94,4 +93,8 @@ enum class Setting(val key: String, val defaultValue: SettingType) {
     LIVERELOAD("enable_livereload", SettingType.BooleanValue(true)),
     PREFIX_BUNDLER("prefix_bundler", SettingType.BooleanValue(true)),
     JEKYLL_ENV("jekyll_env", SettingType.StringValue("development")),
+
+    // Editor
+    EDITOR_THEME("set_theme", SettingType.IntValue(1)),
+    DEBOUNCE_DELAY("debounce_delay", SettingType.FloatValue(1f))
```

Comment:
Please move this below the General settings. Also remove the redundant import for `BuildConfig`.